### ISSUE TITLE
p1 fails with ValueError

### ISSUE THREAD

**Issue by iancrossfield** [kevin218/POET#20]
TLDR: Matplotlib has changed the valid options for imshow.  Commenting out the 'origin' keyword results in a successful execution and avoids the error trace below.

$ python poet.py p1

Traceback (most recent call last):
  File "/home/ianc/proj/spitzer/hd73344/poet/run/poet.py", line 44, in <module>
    e = p1.Event(arg)
  File "/home/ianc/python/POET/code/lib/poet_1event.py", line 44, in __init__
    self.check()
  File "/home/ianc/python/POET/code/lib/poet_1event.py", line 410, in check
    **plt.imshow(image, interpolation='nearest', origin='ll', cmap=plt.cm.gray)**
  File "/home/ianc/miniconda3/lib/python3.9/site-packages/matplotlib/_api/deprecation.py", line 456, in wrapper
    return func(*args, **kwargs)
  File "/home/ianc/miniconda3/lib/python3.9/site-packages/matplotlib/pyplot.py", line 2640, in imshow
    __ret = gca().imshow(
  File "/home/ianc/miniconda3/lib/python3.9/site-packages/matplotlib/_api/deprecation.py", line 456, in wrapper
    return func(*args, **kwargs)
  File "/home/ianc/miniconda3/lib/python3.9/site-packages/matplotlib/__init__.py", line 1412, in inner
    return func(ax, *map(sanitize_sequence, args), **kwargs)
  File "/home/ianc/miniconda3/lib/python3.9/site-packages/matplotlib/axes/_axes.py", line 5482, in imshow
    im = mimage.AxesImage(self, cmap, norm, interpolation,
  File "/home/ianc/miniconda3/lib/python3.9/site-packages/matplotlib/image.py", line 929, in __init__
    super().__init__(
  File "/home/ianc/miniconda3/lib/python3.9/site-packages/matplotlib/image.py", line 249, in __init__
    _api.check_in_list(["upper", "lower"], origin=origin)
  File "/home/ianc/miniconda3/lib/python3.9/site-packages/matplotlib/_api/__init__.py", line 129, in check_in_list
    raise ValueError(msg)
**ValueError: 'll' is not a valid value for origin; supported values are 'upper', 'lower'**
